test/dartdoc_test.dart

```diff
@@ -6,15 +6,40 @@ library dartdoc.dartdoc_test;
 
 import 'dart:io';
 
+import 'package:cli_util/cli_util.dart';
 import 'package:dartdoc/dartdoc.dart';
-import 'package:dartdoc/src/package_utils.dart';
+import 'package:dartdoc/src/model.dart';
+import 'package:dartdoc/src/package_meta.dart';
+import 'package:path/path.dart' as path;
 import 'package:unittest/unittest.dart';
 
+import 'src/utils.dart';
+
 void main() {
-  group('dartdoc test', () {
-    test('version info', () {
-      String version = getPackageVersion(Directory.current.path);
-      expect(version, VERSION);
+  group('dartdoc', () {
+    Directory tempDir;
+
+    setUp(() {
+      tempDir = new Directory(path.join(Directory.systemTemp.path, 'temp'));
+    });
+
+    tearDown(() {
+      delete(tempDir);
+    });
+
+    test('generateDocs', () {
+      PackageMeta meta = new PackageMeta.fromDir(testPackageDir);
+      DartDoc dartdoc =
+          new DartDoc(testPackageDir, [], getSdkDir(), [], tempDir, meta);
+
+      return dartdoc.generateDocs().then((DartDocResults results) {
```

Comment:
So it's a line height issue?

On Sunday, May 17, 2015, Devon Carew <notifications@github.com> wrote:

> In test/dartdoc_test.dart
> <https://github.com/dart-lang/dartdoc/pull/492#discussion_r30473023>:
>
> > +    Directory tempDir;
> > +
> > +    setUp(() {
> > +      tempDir = new Directory(path.join(Directory.systemTemp.path, 'temp'));
> > +    });
> > +
> > +    tearDown(() {
> > +      delete(tempDir);
> > +    });
> > +
> > +    test('generateDocs', () {
> > +      PackageMeta meta = new PackageMeta.fromDir(testPackageDir);
> > +      DartDoc dartdoc =
> > +          new DartDoc(testPackageDir, [], getSdkDir(), [], tempDir, meta);
> > +
> > +      return dartdoc.generateDocs().then((DartDocResults results) {
>
> done -
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dart-lang/dartdoc/pull/492/files#r30473023>.
>


-- 
Sent from my Motorola Startac